client/components/searchBar.jsx

```diff
@@ -48,6 +81,30 @@ class SearchFood extends React.Component {
             </button>
           </div>
         </form>
+        <div>
+          <ul className="search-results">
+            <div className="result-list">
+            {this.state.result.map(item => {
+              return <li key={item.nix_item_id}>
+                <div className="result-image">
+                  <img src={item.photo.thumb} />
+                </div>
+                <div className="description">
+                  <p>{item.food_name}</p>
+                  <p className="calories">Calories: {item.nf_calories.toFixed()}</p>
+                  <p className="serving">Serving: {item.serving_qty.toFixed(1)} {item.serving_unit}</p>
+                </div>
+                <div className="add">
+                <button className="add-item">
+                    <i className="fas fa-plus add-icon" onClick={this.selectItem} data-id={item.food_name} data-calories={item.nf_calories} data-serving={item.serving_qty}></i>
```

Comment:
You should move the `onClick` to the button. A larger click target is much easier for users.